tests/sentry/deletions/test_group.py

```diff
@@ -426,48 +426,3 @@ def test_issue_platform_batching(self, mock_bulk_snuba_queries: mock.Mock) -> No
             assert first_batch == [group2.id, group1.id]  # group2 has less times_seen than group1
             # group3 and group4 have the same times_seen, thus sorted by id
             assert second_batch == [group3.id, group4.id]
-
-    @mock.patch("sentry.deletions.tasks.nodestore.bulk_snuba_queries")
-    def test_issue_platform_batching_with_new_task(
```

Comment:
Is this test going to be relevant again once we reimplement the batching? Should we just skip it for now instead of deleting it?

Reply:
That would not have been bad! I will handle it.